ui/raidboss/data/07-dt/dungeon/strayborough-deadwalk.ts

```diff
@@ -0,0 +1,257 @@
+import Conditions from '../../../../../resources/conditions';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// TODO: Add basically anything to help with the doll charges.
+// TODO: Determine safe spots for Tea Awhirl
+// TODO: Determine safe spots for Toiling Teapots
```

Comment:
Maybe add a TODO for the giant falling ferris wheel (even though it's not part of a boss fight)?

Reply:
Oh, sorry, that's Tilt-a-wheel. I'll explicitly note that it's the Ferris wheel.

Reply:
Oh, I see what you mean. Yes.